src/libslic3r/CMakeLists.txt

```diff
@@ -448,7 +448,7 @@ target_include_directories(libslic3r PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} PUBLIC
 target_include_directories(libslic3r PUBLIC ${EXPAT_INCLUDE_DIRS})
 
 # Find the OCCT and related libraries
-set(OpenCASCADE_DIR "${CMAKE_PREFIX_PATH}/lib/cmake/occt")
+set(OpenCASCADE_DIR "${CMAKE_PREFIX_PATH}/lib/cmake/occt" CACHE PATH "Path that contains OpenCASCADEConfig.cmake")
```

Comment:
This just moves it into the cache so it can be passed on the cmake command-line, there should be no functional change when it's not passed.